packages/react-native-client/src/hooks/usePeers.ts

```diff
@@ -166,10 +166,14 @@ export function usePeers<
   PeerMetadata extends GenericMetadata = GenericMetadata,
   ServerMetadata extends GenericMetadata = GenericMetadata,
 >() {
-  const peers = useFishjamEventState<Peer<PeerMetadata, ServerMetadata>[]>(
+  const peers = useFishjamEventState(
     ReceivableEvents.PeersUpdate,
     RNFishjamClientModule.getPeers<PeerMetadata, ServerMetadata>(),
-    (peersWithoutActive) => addIsActiveToTracks(peersWithoutActive),
+    (peersWithoutActive) =>
+      addIsActiveToTracks(peersWithoutActive) as Peer<
+        PeerMetadata,
+        ServerMetadata
+      >[],
```

Comment:
Super nit: would it be cleaner to add casting to `useFishjamEventState` resutl? (instead of casting 3rd param)

Reply:
This cast is actually not needed at all, I'm not sure why it's here. Thanks!